src/main/java/seedu/address/logic/parser/DeleteCommandParser.java

```diff
@@ -46,7 +46,13 @@ public Command parse(String arguments) throws ParseException {
 
         case DeleteQuestionCommand.COMMAND_WORD:
             ParserUtil.checkEmptyArgument(DeleteQuestionCommand.MESSAGE_USAGE);
-            return new DeleteQuestionCommand(deleteArguments.trim());
+
+            try {
+                int index = Integer.parseInt(deleteArguments.trim());
+                return new DeleteQuestionCommand(index);
+            } catch (NumberFormatException e) {
+                throw new ParseException(INDEX_NOT_A_NUMBER);
```

Comment:
Is it better if the error message is just MESSAGE_UNKNOWN_COMMAND to make it consistent?